### PR TITLE
add sidebar layout

### PR DESCRIPTION
**Pull request by Fari-cloudflight**
### Description

add sidebar layout

### Which Component is affected or generated?

/sidebar/sidebar.component.html
/sidebar/sidebar-item/sidebar-item.component
/content-panel-container-sidebar/content-panel-container-sidebar.component
/content-panel-container/content-panel-container.component

### CODE REVIEW

File: projects/material-addons/src/lib/content-panel/content-panel-container-sidebar/content-panel-container-sidebar.component.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+:host {
+  border-top: 0.05rem solid #cccccc;
```
Comment (diglesias-clf) [inline projects/material-addons/src/lib/content-panel/content-panel-container-sidebar/content-panel-container-sidebar.component.scss:2]: doesn't it make sense to move this color (and ones from the sidebar) to the theme (or other general file)?

File: projects/material-addons/src/lib/sidebar/sidebar.component.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<div class="sidebar-wrapper" [class.collapsed]="collapsed">
+  <div class="sidebar-header">
+    <button mat-button color="basic" (click)="toggleCollapse()"><mat-icon>{{collapsed?'keyboard_double_arrow_right':'keyboard_double_arrow_left'}}</mat-icon></button>
+  </div>
+  <div class="item-list">
+  <ng-content></ng-content>
```
Comment (diglesias-clf) [inline projects/material-addons/src/lib/sidebar/sidebar.component.html]: indentation missing

File: projects/material-addons/src/lib/sidebar/sidebar.component.scss [hunk 3]
```diff
@@ -0,0 +1,48 @@
+$color-gray: #707070;
+$color-border: #cccccc;
+
+.sidebar-wrapper {
+  min-height: calc(100vh - 128px);
```
Comment (diglesias-clf) [inline projects/material-addons/src/lib/sidebar/sidebar.component.scss]: why do we need this? would it also work if there is no flow-bar (header is only 60px)?

File: projects/material-addons/src/lib/sidebar/sidebar.component.scss [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    }
+  }
+
+  .item-list {
+    display: flex;
+    flex-direction: column;
+    margin-top: 16px;
+
+    .item {
+      padding: 9px 24px 8px;
```
Comment (diglesias-clf) [inline projects/material-addons/src/lib/sidebar/sidebar.component.scss]: is this not duplicated with sidebar-item.component.scss and therefore not needed?

**Review by diglesias-clf — CHANGES_REQUESTED** [review]

File: projects/material-addons/src/lib/sidebar/sidebar.component.html (hunk 2 above)
Comment (Fari-cloudflight) [reply projects/material-addons/src/lib/sidebar/sidebar.component.html]: done

File: projects/material-addons/src/lib/sidebar/sidebar.component.scss (hunk 3 above)
Comment (Fari-cloudflight) [reply projects/material-addons/src/lib/sidebar/sidebar.component.scss]: fixed

File: projects/material-addons/src/lib/sidebar/sidebar.component.scss (hunk 4 above)
Comment (Fari-cloudflight) [reply projects/material-addons/src/lib/sidebar/sidebar.component.scss]: removed

File: projects/material-addons/src/lib/content-panel/content-panel-container-sidebar/content-panel-container-sidebar.component.scss (hunk 1 above)
Comment (Fari-cloudflight) [reply projects/material-addons/src/lib/content-panel/content-panel-container-sidebar/content-panel-container-sidebar.component.scss:2]: this is "best practice" in that project:
[content-panel-container-content.component.scss#L4](https://github.com/lukaspoeberl/material-addons/blob/f118cc12f9a6465fdac5fe50b528946b2fda1480/projects/material-addons/src/lib/content-panel/content-panel-container-content/content-panel-container-content.component.scss#L4)

and would require general refactoring in project

**Review by diglesias-clf — APPROVED** [review]

**Pull request merged**